addons/account/views/account_move_views.xml

```diff
@@ -515,6 +515,7 @@
                             invisible="context.get('default_move_type') not in ('out_invoice', 'out_refund', 'out_receipt', 'in_invoice', 'in_refund','in_receipt')"/>
                     </header>
                     <field name="made_sequence_gap" column_invisible="True"/>
+                    <field name="country_code" column_invisible="True"/>
```

Comment:
You don't need to do this anymore starting from v18 :)

Reply:
I remove this code, and it works. It's amazing.